code/script.js

```diff
@@ -1,51 +1,196 @@
+    inputWrapper.innerHTML = "";
+    setTimeout(() => showTypesOptions('Orange'), 500)
+  });
+}; 
+
+// Question 3 
+const showTypesOptions = (flowerType) => {
+  userFormInputs.color = flowerType;
+  showMessage(`I choose ${flowerType}.`, 'user'); 
+
+    setTimeout(() => showMessage(`Great choice! What type of ${flowerType} flowers?`, 'bot'), 1000);
+  
+    setTimeout(() => {
+      if (flowerType === 'Red') {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127801; Select a flower type: </option>
+          <option value="Rose">Rose</option>
+          <option value="Coronation">Coronation</option>
+          <option value="Poppy">Poppy</option>
+        </select>
+      `;
+      } else if (flowerType === 'Yellow') {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127803; Select a flower type:</option>
+          <option value="Sunflower">Sunflower</option>
+          <option value="Daffodil">Daffodil</option>
+          <option value="Daisies">Daisies</option>
+        </select>
+      `;
+      } else {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127809; Select a flower type:</option>
+          <option value="Dahlia">Dhalia</option>
+          <option value="Tulip">Tulip</option>
+          <option value="Lily">Lily</option>
+        </select>
+      `;
+      }
+      const select = document.getElementById('select');
+      select.addEventListener('change', () => {
+        inputWrapper.innerHTML = "";
+        setTimeout(() => bouquetSize(select.value), 1000);
+      });
+    }, 1500);
+  };
+
+  // Question 4
+    const bouquetSize = (select) => {
+      userFormInputs.size = select;
+      showMessage(`I pick ${select}.`, 'user');
+      setTimeout(() => showMessage(`Sweet! What size of bouquet for those ${select}?`, 'bot'), 1000);
+    
+      setTimeout(() => {
+        inputWrapper.innerHTML = `
+        <button id="small">Small</button>
+        <button id="medium">Medium</button>
+        <button id="large">Large</button>
+        `;
+    
+        document.getElementById('small').addEventListener('click', () => {
+          inputWrapper.innerHTML = "";
+          setTimeout(() => paymentSize('Small'), 1000);
+        });
+        document.getElementById('medium').addEventListener('click', () => {
+          inputWrapper.innerHTML = "";
+          setTimeout(() => paymentSize('Medium'), 1000);
+        });
+        document.getElementById('large').addEventListener('click', () => {
+          inputWrapper.innerHTML = "";
+          setTimeout(() => paymentSize('Large'), 1000);
+        });
+      }, 1500);
+    };
+
+      const paymentSize = (size) => {
+        userFormInputs.payment = size;
+        showMessage(`I want a ${size}.`, 'user');
+        
+        if (size === 'Small') {
+          price = '150kr';
```

Comment:
Wow! What you did here with the global variable `userFormInputs` and passing on values on that with connection to the parameter is super advanced. Would love to learn to do that too!